packages/typespec-autorest/test/flatten.test.ts

```diff
@@ -0,0 +1,66 @@
+import { deepStrictEqual } from "assert";
+import { describe, it } from "vitest";
+import { openApiFor } from "./test-host.js";
+
+describe("typespec-autorest: flatten", () => {
```

Comment:
now that we moved to vitest we don't actually need this top level describe, it just adds another level of nesting in the UI too